pages/index.tsx

```diff
@@ -12,38 +10,8 @@ export default function Home() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
```

Comment:
Do you think these lines of code should only be inside the index page or maybe somewhere globally?

If you want to fix this and think this is a problem, create a new card in Zenhub and assign it to yourself after you have finished this hero section card.